lib/models/store_transaction.dart

```diff
@@ -9,10 +9,14 @@ part 'store_transaction.g.dart';
 class StoreTransaction with _$StoreTransaction {
   const factory StoreTransaction(
     /// RevenueCat Id associated to the transaction.
+    @JsonKey(name: 'transactionIdentifier') String transactionIdentifier,
+
+    /// Deprecated: Use transactionIdentifier instead.
+    @Deprecated('Use transactionIdentifier instead.')
     @JsonKey(name: 'revenueCatId') String revenueCatIdentifier,
 
     /// Product Id associated with the transaction.
-    @JsonKey(name: 'productId') String productIdentifier,
+    @JsonKey(name: 'productIdentifier') String productIdentifier,
```

Comment:
`productId` was also deprecated, but since flutter was using `productIdentifier`, devs shouldn't see any changes. I only changed the mapping to use the new name.